packages/web3/package.json

```diff
@@ -34,7 +34,7 @@
   "author": "Alephium dev <dev@alephium.org>",
   "config": {
     "alephium_version": "3.9.0",
-    "explorer_backend_version": "2.2.6"
```

Comment:
Is this change relevant in this PR? Usually the version is updated when we want to update the [explorer backend API file](https://github.com/Benjtalkshow/alephium-web3/blob/test-coverage-for-web3/src/utlis/packages/web3/src/api/api-explorer.ts), which doesn't seem to be the case here.